src/fastDeepEqualJson.ts

```diff
@@ -1,6 +1,11 @@
 const isArray = Array.isArray;
 const hasProp = Object.prototype.hasOwnProperty;
 
+/**
+ * Determine if two objects are deeply equal.
```

Comment:
maybe include that this is faster than `fastDeepEqual` so this should be used instead for compatible types?

Reply:
Will do. I should say, it's faster and the speed matters for _large_ objects, but it's a trivial difference for small objects.